spec/api/comment_thread_spec.rb

```diff
@@ -477,6 +477,18 @@ def test_unicode_data(text)
         check_thread_result_json(nil, thread, parsed)
       end
 
+      context 'when requesting the thread for informational purposes' do
+        subject do
+          get "/api/v1/threads/#{thread.id}", with_responses: false # we're asking for no responses here.
```

Comment:
Glad to see that PR.  However, I was referring to the issue of extra calls you had commented on here: 
https://github.com/edx/cs_comments_service/pull/205#issuecomment-242734427